heltour/tournament/tests/test_login.py

```diff
@@ -3,7 +3,8 @@
 from django.test import TestCase
 from unittest.mock import patch
 from heltour.tournament import oauth
-from .testutils import *
+from .testutils import *#
```

Comment:
generally speaking it's worth  being explicit about your imports. from a practical perspective it helps static analysis because when you have `import *` the program (it it's not like a LSP or something) really struggles to know what variables/functions etc are on the stack.

Additionally relative imports are the devil, prefere absolute imports if possible.